aiolifx/fixtures/fixtures.py

```diff
@@ -0,0 +1,51 @@
+from dataclasses import dataclass
+from aiolifx.fixtures.chain_light import ChainLightMixin
+from aiolifx.fixtures.color_light import ColorLightMixin
+from aiolifx.fixtures.hev_light import HevLightMixin
+from aiolifx.fixtures.light import LightMixin
+from aiolifx.fixtures.multizone_light import MultizoneLightMixin
+from aiolifx.fixtures.switch import SwitchMixin
+from aiolifx.fixtures.matrix_light import MatrixLightMixin
+from aiolifx.products import products_dict
+
+
+class Light(LightMixin):
+    pass
+
+class ColorLight(LightMixin, ColorLightMixin):
+    pass
+
+class MultizoneLight(LightMixin, ColorLightMixin, MultizoneLightMixin):
+    pass
+
+class HevLight(LightMixin, HevLightMixin):
+    pass
+
+class ChainLight(LightMixin, MatrixLightMixin, ChainLightMixin):
+    pass
+
+class MatrixLight(LightMixin, MatrixLightMixin):
+    pass
+
+class Switch(SwitchMixin):
+    pass
+
+
+def get_fixture(product_id, req_with_resp, req_with_ack, fire_and_forget):
+    product = products_dict[product_id]
+    if product.relays and product.buttons:
+        return Switch(req_with_resp, req_with_ack, fire_and_forget)
+    if product.multizone and product.extended_multizone:
+        return MultizoneLight(req_with_resp, req_with_ack, fire_and_forget)
+    if product.color:
+        return ColorLight(req_with_resp, req_with_ack, fire_and_forget)
+    if product.infrared and product.hev:
+        return HevLight(req_with_resp, req_with_ack, fire_and_forget)
+    if product.chain:
+        return ChainLight(req_with_resp, req_with_ack, fire_and_forget)
```

Comment:
We can combine Matrix and Chain into a single Mixin as all Matrix devices have a Chain, though most only have 1 tile on their chain. I'm using lowercase on purpose as there is also the LIFX Tile which can have up to 5 tiles on its chain, with each tile being a LIFX Tile. 

The names used for elements by the protocol definition hasn't really adapted to the reality that the device after which most elements were named is now discontinued and current use doesn't match that previous naming well at all. 😂